cocos/platform/CCPlatformConfig.h

```diff
@@ -154,6 +154,7 @@ THE SOFTWARE.
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
 #ifndef __MINGW32__
 #pragma warning (disable:4127) 
+#pragma warning (disable:4018)
```

Comment:
@lbfamous  

 signed/unsigned mismatch would make our code base error-prone, I think we should fix the issue rather than disable the warning.
@minggo  Thoughts?